.github/workflows/generator.yaml

```diff
@@ -1,6 +1,9 @@
 name: Client Generator
 
-on: [push]
+on: [push, pull_request]
```

Comment:
I like your changes over #22, but I also do not like to mix CI and code changes.

Can we split these?

Reply:
Opened: https://github.com/googleapis/google-cloud-rust/pull/26